app/controllers/verify/confirmations_controller.rb

```diff
@@ -30,8 +30,17 @@ def finish_proofing_success
       @recovery_code = recovery_code
       idv_session.complete_profile
       idv_session.recovery_code = nil
-      flash[:allow_confirmations_continue] = true
+      create_account_verified_event
       flash.now[:success] = t('idv.messages.confirm')
+      flash[:allow_confirmations_continue] = true
+    end
+
+    def create_account_verified_event
+      no_verified_event = current_user.events.find_by(
+        event_type: Event.event_types['account_verified']
+      ).empty?
+
+      create_user_event(:account_verified) if no_verified_event
```

Comment:
Totally agree we should test it -- my comment was that I wasn't sure how to test it. Sorry if that was unclear! 

I was under the impression that we don't test private methods directly?

When I try calling `get :index` twice in the controller spec, I get undefined method `encrypted' for nil:NilClass` from the `encrypt_recovery_pii` method in the profile model. Since I can refresh the page in browser and not get an error, I'm thinking the way I'm trying to test a repeated action is wrong.